app/uk/gov/hmrc/leakdetection/model/Summary.scala

```diff
@@ -52,7 +52,7 @@ object RepositorySummary {
 
 case class BranchSummary(
                           branch: String,
-                          reportId: ReportId,
+                          reportId: String,
```

Comment:
Should we keep this as `ReportId`?

Reply:
We have value classes for Repository, Branch, ReportId & RuleId but they aren't always used.  I changed this so it was least consistent to this class.  If value classes are preferred, then I think they should be changed everywhere.  This would touch a lot of files so would perhaps be better on a separate PR.